docs/reference/logging-levels.md

```diff
@@ -1,9 +1,9 @@
 (reference-logging-levels)=
 # Logging levels
 
-> See also: [Configure Multipass’s default logging level](/how-to-guides/customise-multipass/configure-multipass-default-logging-level)
+> See also: [Configure Multipass’s default logging level](/how-to-guides/customise-multipass/configure-multipass-default-logging-level), [How to access logs](/how-to-guides/troubleshoot/access-logs)
 
-In Multipass, a hierarchy of logging levels is used is used to convey severity and improve visibility of important events. Multipass uses the following levels ranked from most severe to least severe for its background daemon and child processes.
+In Multipass, a hierarchy of logging levels is used to convey severity and improve visibility of important events. Multipass uses the following levels ranked from most severe to least severe for its background daemon and child processes.
```

Comment:
maybe add comas for clarity:
Multipass uses the following levels, ranked from most severe to least severe, for its background daemon and child processes